src/bios_handler.cpp

```diff
@@ -143,6 +143,9 @@ void IbmBiosHandler::backUpOrRestoreBiosAttributes()
 {
     // process FCO
     processFieldCoreOverride();
+
+    // process memory mirror mode
+    processMemoryMirrorMode();
```

Comment:
processAutoMemoryMirror
as the keyword is AMM like FCO

Reply:
Done.